app/assets/stylesheets/application/common.scss

```diff
@@ -266,3 +268,66 @@ header .container-fluid nav.navbar li.dropdown {
     line-height: 1.5;
   }
 }
+.body-content {
+  width: 92%;
```

Comment:
why 92%? It's super random number isn't it?

Reply:
Actually no it's not. First I was doing a prototype (https://github.com/LucasAndrad/LearnItGirl-Prototype) and this "body-content width" should be 90%, but when I pass the prototype code to LearnItGirl the content got a little smaller, I tried to figure it out what was causing that but I didn't found, so width with 92% solved this problem.  Of course if you want can be 90%, no problem at all.